pubspec.yaml

```diff
@@ -9,4 +9,4 @@ description: >
   This library declares readable symbolic names for character codes.
 homepage: https://github.com/dart-lang/charcode
 environment:
-  sdk: '>=1.0.0 <2.0.0'
+  sdk: '>=1.0.0 <2.0.0-dev.infinity'
```

Comment:
Consider just removing the constraint entirely.
If we change the language enough to break `const int id = integerLiteral;`, then we have bigger problems than this package being considered compatible.